pydeseq2/dds.py

```diff
@@ -432,11 +440,27 @@ def vst_transform(self, counts: Optional[np.ndarray] = None) -> np.ndarray:
                 f"Found fit_type '{self.fit_type}'. Expected 'parametric' or 'mean'."
             )
 
-    def deseq2(self) -> None:
+    def deseq2(self, fit_type: Optional[Literal["parametric", "mean"]] = None) -> None:
         """Perform dispersion and log fold-change (LFC) estimation.
 
         Wrapper for the first part of the PyDESeq2 pipeline.
+
+
+        Parameters
+        ----------
+        fit_type : str
+            Either None, ``"parametric"`` or ``"mean"`` for the type of fitting of
+            dispersions to the mean intensity.``"parametric"``: fit a dispersion-mean
+            relation via a robust gamma-family GLM. ``"mean"``: use the mean of
+            gene-wise dispersion estimates.
+
+            If None, the fit_type provided at class initialization is used.
+            (default: ``None``).
         """
+        if fit_type is not None:
+            self.current_fit_type = fit_type
+            print(f"Using {self.current_fit_type} fit type.")
```

Comment:
`self.current_fit_type` doesn't seem to be used elsewhere, so the `deseq2()` `fit_type` argument doesn't have any effect